sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/models/traits/HttpTrait.java

```diff
@@ -5,17 +5,18 @@
 
 import com.generic.core.http.client.HttpClient;
 import com.generic.core.http.models.HttpLogOptions;
+import com.generic.core.http.models.HttpRedirectOptions;
 import com.generic.core.http.models.HttpRetryOptions;
 import com.generic.core.http.pipeline.HttpPipeline;
 import com.generic.core.http.pipeline.HttpPipelinePolicy;
 
 /**
- * An {@link com.generic.core.models.traits SDK for Java trait} providing a consistent interface for configuration
- * of HTTP-specific settings. Refer to the SDK for Java
+ * An {@link com.generic.core.models.traits SDK for Java trait} providing a consistent interface for configuration of
+ * HTTP-specific settings. Refer to the SDK for Java
```

Comment:
Wording is a bit confusing here, and likely the same in `azure-core` too.

```suggestion
 * A {@link com.generic.core.models.traits trait} providing a consistent interface for configuration of
 * HTTP-specific settings. Refer to the SDK for Java
```

I don't think we need to, or should, include `SDK for Java`, it's a bit redundant.